src/DoubleLinkedList.js

```diff
@@ -0,0 +1,124 @@
+const IList = require('./IList');
+
+const Node = function(value) {
+    this.data = value;
+    this.next = null;
+    this.previous = null;  //uzel sozdaniye
+};
+
+const DoubleLinkedList =  function() {
+    IList.apply(this, arguments); // putevoditel kot napravlayt
+    this._root = null; // nacalo
+    this._tail = null; //konec
+    this._size = 0; 
+};
+
+DoubleLinkedList.prototype = Object.create(IList.prototype);
+DoubleLinkedList.prototype.constructor = DoubleLinkedList;
+
+DoubleLinkedList.prototype.clear = function() {
+    this._size = 0;
+    this._tail = null;
+    this._root = null;
+};
+
+DoubleLinkedList.prototype.init = function(value) {
+      for (let i = 0; i < value.lenght; i++) {
+          let node = new Node(value[i]);
+
+          !this._root && (this._root = node);
+          node._tail = this._tail;
```

Comment:
Node doesn't have _tail
By your own interface it has _data, _next and _previous
but should have _value _next and _prev